ui/src/components/EntryDetailed/EntrySections.tsx

```diff
@@ -167,7 +169,7 @@ export const EntryBodySection: React.FC<EntryBodySectionProps> = ({
 
     return <React.Fragment>
         {content && content?.length > 0 && <EntrySectionContainer
-                                                title='Body'
+                                                title={title}
```

Comment:
this is the same change the you made in https://github.com/up9inc/mizu/pull/644, right?
if you merge the redis one before this will not be part of this change IMO

Reply:
@gadotroee doesn't matter. It required the same thing. We'll resolve the conflicts if any.